apps/studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -161,35 +153,40 @@ export const ProjectList = ({ organization: organization_, rewriteHref }: Projec
       <Card className="flex-1 min-h-0 overflow-y-auto mb-8" onScroll={handleScroll}>
         <Table>
           {/* [Joshen] Ideally we can figure out sticky table headers here */}
-          <TableHeader className="[&>tr>th]:sticky [&>tr>th]:top-0">
+          <TableHeader>
             <TableRow>
-              <TableHead>Project</TableHead>
-              <TableHead>Status</TableHead>
-              <TableHead>Compute</TableHead>
-              <TableHead>Region</TableHead>
-              <TableHead>Created</TableHead>
-            </TableRow>
-            <TableRow className="!border-b-0">
-              <TableCell colSpan={5} className="p-0">
-                <LoadingLine loading={isFetching} />
-              </TableCell>
+              <TableHead className={noResults && 'text-foreground-muted'}>
+                Project
+              </TableHead>
+              <TableHead className={noResults && 'text-foreground-muted'}>
+                Status
+              </TableHead>
+              <TableHead className={noResults && 'text-foreground-muted'}>
+                Compute
+              </TableHead>
+              <TableHead className={noResults && 'text-foreground-muted'}>
+                Region
+              </TableHead>
+              <TableHead className={noResults && 'text-foreground-muted'}>
+                Created
+              </TableHead>
             </TableRow>
           </TableHeader>
           <TableBody>
             {noResultsFromStatusFilter ? (
-              <TableRow>
-                <TableCell colSpan={5} className="p-0">
+              <TableRow className="[&>td]:hover:bg-inherit">
```

Comment:
Prevent a hover state if there are no actual rows, as per [design system](https://supabase-design-system.vercel.app/design-system/docs/ui-patterns/empty-states#zero-results).

Reply:
Would be cool to systemise this somehow.